codice/ddf
# DDF-3222 Implemented cache busting for all DDF ui modules.

#### What does this PR do?

Implements cache busting for all DDF ui modules.

#### Who is reviewing it? 

@garrettfreibott 
@tbatie 

#### Select relevant component teams: 

@codice/ui

#### Choose 2 committers to review/merge the PR. 

@andrewkfiedler 
@pklinef

#### How should this be tested?

Perform the following in Chrome, Firefox, IE with any devtools closed:

- open any ui and ensure that everything still loads correctly
- make some changes to that ui's source code
- rebuild that ui module and hot deploy its bundle
- ensure that the changes are loaded without explicit user actions (clearing the cache)

**NOTE:** this PR will only work when #1841 is merged, so for testing you must manually apply the changes in #1841.

#### What are the relevant tickets?

[DDF-3222](https://codice.atlassian.net/browse/DDF-3222)

#### Checklist:


#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
